k8s/namespaces/fees-pay/fees-register-frontend/demo.yaml

```diff
@@ -3,28 +3,14 @@ apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
   name: fees-register-frontend
-  namespace: fees-pay
   annotations:
     flux.weave.works/automated: "true"
```

Comment:
Is this needed?

Reply:
yes i have removed the **namespace: fees-pay** as its present in here
spec:
  releaseName: fees-register-frontend

Reply:
I dont think below is needed?

```
flux.weave.works/automated: "true"
 flux.weave.works/tag.nodejs: pr-401-*
```